### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by jainaman224** [jainaman224/Algo_Ds_Notes#202]
Add documentation to the folders where there is no README.md file.

**Comment by oanarosca** [comment]
I want to work on this :smile:

**Comment by jainaman224** [comment]
@oanarosca Thank you. It will be better if you open new pull request for each README else you can add upto 5 README in one PR.

**Comment by seancyw** [comment]
Hi, I would like to work on this too :)

**Comment by jainaman224** [comment]
@seancyw @oanarosca  Please specify the topic you want to work on and proceed.

**Comment by paliitr** [comment]
Working on documentation for stack implementations.

**Comment by PiyushGoyal443** [comment]
Can i also work for following 
1.Stack_using_lists
2. Stack_using_array
3. Breadth First Search

**Comment by paliitr** [comment]
@PiyushGoyal443 I was working on 1. and 2. Please see previous messages before starting work as it saves time for both. Anyways good job on the readme.
